### PR TITLE
[DO NOT MERGE] Add Nano workaround (proper)

### PR DESCRIPTION
**Pull request by blackdwarf**
Updating the Nano workaround.

skip ci please

/cc @leecow @gkhanna79 @MichaelSimons

### CODE REVIEW

**Comment by blackdwarf** [comment]
Please don't merge this. 😄

File: Documentation/known-issues.md [hunk 1]
```diff
@@ -29,11 +29,13 @@ You can verify whether you have the right version using the  `openssl version` c
...
- 
-If you use “dotnet publish”, make sure to copy all DLL files from c:\windows\system32\forwarders to your publish directory as well.
- 
-If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, you will need to copy forwarders' DLLs due to a bug in the forwarders' resolve logic (the bug has been fixed in later releases). The forwarders' location is in `c:\windows\system32\forwarders`. The destination depends on the [type of deployment](https://dotnet.github.io/docs/core-concepts/app-types.html) that you are choosing for your application. 
+
+For portable applications, the forwarders need to be copied to the shared runtime location. The shared runtime can be found wherever you installed .NET Core (by default, it is `C:\Program Files\dotnet`) under the following path: `shared\Microsoft.NETCore.App\<version>\`. 
+
+For self-contained applications, the forwarders need to be copied over into the application folder, that is, whereever you put the published output.
```
Comment (eerhardt) [inline Documentation/known-issues.md]: typo `whereever`

File: Documentation/known-issues.md [hunk 2]
```diff
@@ -29,11 +29,13 @@ You can verify whether you have the right version using the  `openssl version` c
 
 ## Running .NET Core CLI on Nano Server
 
-If you’re using Nano Server Technical Preview 5 with .NET Core CLI, you will need to copy all DLL files from c:\windows\system32\forwarders to c:\windows\system32, due to a bug that has since been fixed in later releases.
- 
-If you use “dotnet publish”, make sure to copy all DLL files from c:\windows\system32\forwarders to your publish directory as well.
- 
-If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, you will need to copy forwarders' DLLs due to a bug in the forwarders' resolve logic (the bug has been fixed in later releases). The forwarders' location is in `c:\windows\system32\forwarders`. The destination depends on the [type of deployment](https://dotnet.github.io/docs/core-concepts/app-types.html) that you are choosing for your application.
```
Comment (gkhanna79) [inline Documentation/known-issues.md]: I would suggest phrasing this as:

`...you will need to copy binaries from c:\windows\system32\forwarders. The destination...`

Also, given that we are testing on Nano TP5, please ensure the same is reflected in supported platforms.

Comment (blackdwarf) [inline Documentation/known-issues.md]: Good point. :)

File: Documentation/known-issues.md (hunk 1 above)
Comment (blackdwarf) [inline Documentation/known-issues.md]: Hm, [this](https://www.bing.com/search?q=whereever&go=Submit&qs=n&form=QBLH&pq=whereever&sc=8-8&sp=-1&sk=&cvid=D3E40B8859F040A6908C2EDB39E9057C) is telling me it is a valid word?

**Comment by MichaelSimons** [comment]
:shipit:

File: Documentation/known-issues.md (hunk 1 above)
Comment (eerhardt) [inline Documentation/known-issues.md]: There isn't a double 'ee' in it.  You have it spelled correctly above:

```
The shared runtime can be found wherever you installed .NET Core
```

But here there is an extra 'e':

```
that is, whereever you put the published output.
```

File: Documentation/known-issues.md [hunk 3]
```diff
@@ -29,11 +29,13 @@ You can verify whether you have the right version using the  `openssl version` c
 
 ## Running .NET Core CLI on Nano Server
 
-If you’re using Nano Server Technical Preview 5 with .NET Core CLI, you will need to copy all DLL files from c:\windows\system32\forwarders to c:\windows\system32, due to a bug that has since been fixed in later releases.
- 
-If you use “dotnet publish”, make sure to copy all DLL files from c:\windows\system32\forwarders to your publish directory as well.
- 
-If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, due to a bug in the product, you will need to copy binaries from  `c:\windows\system32\forwarders`. The destination depends on the [type of deployment](https://dotnet.github.io/docs/core-concepts/app-types.html) that you are choosing for your application.
```
Comment (naamunds) [inline Documentation/known-issues.md]: It's not clear if `a bug in the product` refers to a bug in Nano or a bug in .NET.

File: Documentation/known-issues.md [hunk 4]
```diff
@@ -29,11 +29,13 @@ You can verify whether you have the right version using the  `openssl version` c
...
- 
-If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, due to a bug in the product, you will need to copy binaries from  `c:\windows\system32\forwarders`. The destination depends on the [type of deployment](https://dotnet.github.io/docs/core-concepts/app-types.html) that you are choosing for your application. 
+
+For portable applications, the forwarders need to be copied to the shared runtime location. The shared runtime can be found wherever you installed .NET Core (by default, it is `C:\Program Files\dotnet`) under the following path: `shared\Microsoft.NETCore.App\<version>\`. 
+
+For self-contained applications, the forwarders need to be copied over into the application folder, that is, whereever you put the published output.
+
+This process will ensure that that the `dotnet` host finds the appropriate APIs it needs.  If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
```
Comment (naamunds) [inline Documentation/known-issues.md]: Typo: `that that`

Comment (naamunds) [inline Documentation/known-issues.md]: Consider writing "forwarders" instead of "DLLs" for clarity.

**Comment by naamunds** [comment]
LGTM, other than a few minor comments (and Eric's comment which is now hidden but appears to still apply).

File: Documentation/known-issues.md (hunk 1 above)
Comment (blackdwarf) [inline Documentation/known-issues.md]: Ehhhhh...fixed. 😄

**Comment by blackdwarf** [comment]
Done.

**Comment by naamunds** [comment]
Looks good now. :shipit:

**Comment by blackdwarf** [comment]
Closing this down as the wrong branch is selected. Will create another PR against `rel/1.0.0`.

**Pull request closed without merging**